components/email-input.jsx

```diff
@@ -39,14 +39,27 @@ var EmailInput = React.createClass({
     });
     this.props.onChange(this.props.name, this);
   },
-  render: function() {
+  renderHint: function() {
     var hintClassIconName = "fa fa-question-circle hint";
     var hintClassName = "hint-msg small";
+    var info = this.props.info;
     if (this.state.showHint) {
       hintClassIconName += " on";
     } else {
       hintClassName += " hidden";
     }
+    if (info) {
```

Comment:
Yeah, singup doesn't have an info button.

It's the little "?" next to the email box in sequential, but I didn't see the need for it to always be there.

That said, if we want, we can create an info box for signup too, but would need new copy for the info. For now I'm fine with the way it is on prod.